modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaScreenContributorClassCheck.testjava

```diff
@@ -0,0 +1,66 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import com.liferay.portal.settings.configuration.admin.display.PortalSettingsConfigurationScreenFactory;
```

Comment:
@nicolas-sss, is this really necessary?

Theoretically, this import is already in the Java class since the original Java class implements the interface `PortalSettingsConfigurationScreenContributor`, therefore add it here shouldn't be necessary. Does it make sense to you?

Reply:
This new import it's about the factory used by the new class. Isn't the Contributor of the inner class. But your comment made me notice that it's missing another import. I will add the one that is missing.